content/python/concepts/statsmodels/statsmodels.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: 'Statsmodels'
+Description: 'The statsmodels library is used to estimate statistical models and perform statistical tests in Python.' 
```

Comment:
We should rephrase it just a little to make it clearer and easier to understand.

```suggestion
Description: 'The statsmodels library is used to estimate the statistical models and perform statistical tests.' 
```